R/bootstrap.nlsfit.R

```diff
@@ -471,6 +498,24 @@ bootstrap.nlsfit <- function(fn,
 
   errors <- apply(par.boot[rr, , drop=FALSE], 2, error, na.rm = TRUE)
 
+  if(is.null(priors$param)==FALSE && is.null(priors$p)==FALSE && is.null(priors$psamples)==FALSE){
+    lp <- length(priors$p)
+    y <- head(y,-lp)
```

Comment:
I see why this is needed. Perhaps it would be better to introduce some other `y` with the priors, like `yp` and `dydp`?